src/libraries/System.Data.Common/src/System/Data/Common/DbException.cs

```diff
@@ -8,6 +8,11 @@ namespace System.Data.Common
     [System.Runtime.CompilerServices.TypeForwardedFrom("System.Data, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089")]
     public abstract class DbException : System.Runtime.InteropServices.ExternalException
     {
+        /// <summary>
+        /// When overriden in a derived class return true in all cases where a simple retry of the operation without any other change may be successful.
+        /// </summary>
+        public virtual bool IsTransient { get; protected set; }
```

Comment:
This shouldn't be an auto-property, but rather a simple getter-only property. This allows implementations greater freedom in how they implement this, and doesn't add a backing field which usually won't be needed. So we need something like:

```c#
public virtual bool IsTransient => false;
```

Reply:
Thanks @roji when the API is approved I will make these changes.